packages/context/src/value-promise.ts

```diff
@@ -28,7 +28,8 @@ export type BoundValue = any;
  */
 export type ValueOrPromise<T> = T | PromiseLike<T>;
 
-export type MapObject<T> = {[name: string]: T};
+// tslint:disable-next-line:no-any
+export type MapObject<T = any> = {[name: string]: T};
```

Comment:
AFAICT, this change is not necessary - I don't see any place in this pull request using `MapObject` without providing `T` value.

Please revert.